package ru.deepthreads.rest.exceptions.other

class AlreadyAMember : Exception()